volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml

```diff
@@ -1,55 +1,73 @@
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```

Comment:
how did you generate the yaml files? are you sure it passes correctly? because it looks like you skipped the `hack/update_labels_in_crd.sh` stage in the generation

Reply:
done